resources/definitions/inat_proton_x_rail.def.json

```diff
@@ -0,0 +1,31 @@
+{
+
+    "name": "Proton X Rail",
+    "version": 2,
+    "inherits": "inat_base",
+    "metadata": {
+        "visible": true,
+        "quality_definition": "inat_base",
+        "platform": "inat_proton_x_buildplate.stl",
+        "platform_offset": [ 0, -0.8, 0]
+    },
+    "overrides": {
+        "machine_name": { "default_value": "Proton X Rail" },
+        "machine_width": { "default_value": 304 },
+        "machine_depth": { "default_value": 304 },
```

Comment:
On the website you claim a size of 300x300mm but the build volume in Cura is 304x304mm in X/Y.

I trust that this is intentional, since:
* The build volume is the same in PrusaSlicer: https://github.com/prusa3d/PrusaSlicer/commit/49f7d11940888edcc57dc54645d9d55200e7287e
* They are not far off.
* You claim that you are member of INAT. I can't verify that claim but the difference will be somewhat innocuous. I trust that you know the limits of the printer better than your marketing department.